### ISSUE TITLE
Missing dependency after install

### ISSUE THREAD

**Issue by wbiker** [Nordaaker/convos#413]
After installing via 
> curl https://convos.by/install.sh | sh -

I tried to start it as recommented with:
> convos/script/convos daemon --listen http://*:8000

But that just yield following error:
`Is is not possible to start Convos at this point, since
there are some missing dependencies that need to be installed:

-  JSON::Validator 3.16 # Can't locate IO/Compress/Gzip.pm
-  LinkEmbedder 1.08 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojo::IOLoop::ForkCall 0.17 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojo::IRC 0.45 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojolicious 8.25 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojolicious::Plugin::OpenAPI 2.18 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require
-  Mojolicious::Plugin::Webpack 0.12 # Attempt to reload Mojo/Base.pm aborted.
Compilation failed in require

Run "convos/script/convos install" to install the missing dependencies above`

Needless to say that the command above not solved the issue.
I fixed it with manual install:
> /usr/bin/perl /home/wolf/convos/script/cpanm -n -l /home/wolf/convos/script/../local -M https://cpan.metacpan.org IO::Compress::Gzip

**Comment by jhthorsen** [comment]
Which version of Perl do you have, and which operating system are you on?

```
$ corelist IO::Compress::Gzip
IO::Compress::Gzip was first released with perl v5.9.4
```

Would you happen to be on some sort of Redhat flavor?

**Comment by wbiker** [comment]
Ah, sorry. That must be an issue with my system then. Recently I messed around with some perl modules because of an updated Perl version. Looks like I caused this issue by myself.

> perl -v
This is perl 5, version 30, subversion 0 (v5.30.0) built for x86_64-linux-thread-multi
I am using Arch: 
> uname -a
5.3.8-arch1-1 #1 SMP PREEMPT @1572357769 x86_64 GNU/Linux
